examples/kubevirt-disk-uploader-tekton.yaml

```diff
@@ -156,8 +156,11 @@ metadata:
   name: kubevirt-disk-uploader-task
 spec:
   params:
+  - name: EXPORT_SOURCE_KIND
+    description: The name of the export source kind
+    type: string
   - name: POD_NAME
-    description: The name of the virtual machine
+    description: The name of the export source
     type: string
   - name: VOLUME_NAME
     description: The volume name of the virtual machine
```

Comment:
if user defines PVC as a source kind, will user has to define this variable as well?

Reply:
I'll check it.

Reply:
Yes, user will have to define this variable, because VMExport still populates volumes field (though there is a single volume) when exporting a disk from PVC.

Reply:
It looks like this:
```
kubevirt-disk-uploader --export-source-kind pvc \
                                         --export-source-name example-pvc \
                                         --volumename example-pvc \
                                         --imagedestination quay.io/org/repo:latest
```